app/src/lib/feature-flag.ts

```diff
@@ -14,7 +14,10 @@ function enableDevelopmentFeatures(): boolean {
     return true
   }
 
-  if (process.env.GITHUB_DESKTOP_PREVIEW_FEATURES) {
+  if (
+    process.env.GITHUB_DESKTOP_PREVIEW_FEATURES !== undefined &&
+    process.env.GITHUB_DESKTOP_PREVIEW_FEATURES === '1'
```

Comment:
This can just be `process.env.GITHUB_DESKTOP_PREVIEW_FEATURES === '1'` — if it’s `'1'`, it’s not `undefined`.